gpytorch/variational/variational_strategy.py

```diff
@@ -73,11 +73,13 @@ class VariationalStrategy(_VariationalStrategy):
         https://www.repository.cam.ac.uk/handle/1810/278022
     """
 
-    def __init__(self, model, inducing_points, variational_distribution, learn_inducing_locations=True):
+    def __init__(
+        self, model, inducing_points, variational_distribution, learn_inducing_locations=True, jitter_val=1e-4
+    ):
         super().__init__(model, inducing_points, variational_distribution, learn_inducing_locations)
         self.register_buffer("updated_strategy", torch.tensor(True))
         self._register_load_state_dict_pre_hook(_ensure_updated_strategy_flag_set)
-
+        self.jitter_val = jitter_val
```

Comment:
```suggestion
        if jitter_val is None:
            self.jitter_val = cholesky_jitter.value()
        else:
            self.jitter_val = jitter_val
```

Reply:
This way we can default to the setting that we use for exact GPs, which the user can specify with a context manager.

(We'll also have to modify line 22 to import `cholesky_jitter`:

```python
from ..settings import _linalg_dtype_cholesky, cholesky_jitter, trace_mode
```

I can't seem to do this through "suggested changes" though... ☹️)